misc/distributed_search/outer.sh

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/env bash
+
+#SBATCH --job-name=dist_AD
+#SBATCH --time=21-00:00:00
+#SBATCH --output=./logs/%j-%x-slurm.out
+
+# slurm parameters
```

Comment:
Encountered a problem here - the prediction of spectral libraries and the generation of the MBR library deliberately let AlphaDIA fail when the library is done, since there are zero rawfiles submitted. To my knowledge, AlphaDIA does not have a "library only" mode yet?

Reply:
I see .. 
"library only" mode is clearlt out of scope of this PR
=> https://github.com/orgs/MannLabs/projects/12/views/1?filterQuery=libr&pane=issue&itemId=87041704

as a quick fix, you can 
`set +e` before the alphadia command that fails,
and 
`set -e` afterwards